linux-anvil-ppc64le-cuda/Dockerfile

```diff
@@ -55,27 +55,29 @@ ADD https://loripsum.net/api /opt/docker/etc/gibberish
 # Naming convention changed with cos8 - see:
 # * https://lists.centos.org/pipermail/centos-devel/2019-September/017847.html
 # * https://www.centos.org/keys/#project-keys
-RUN if [ "$DISTRO_VER" -le "7" ]; then \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${DISTRO_VER} && \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-${DISTRO_VER}-ppc64le; \
-    else \
+RUN if [ "${DISTRO_NAME}${DISTRO_VER}" = "centos7" ]; then \
+        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-AltArch-7-ppc64le; \
+    elif [ "${DISTRO_NAME}${DISTRO_VER}" = "centos8" ]; then \
         rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial; \
+    elif [ "${DISTRO_NAME}${DISTRO_VER}" = "ubi8" ]; then \
+        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release; \
     fi
 
-# Remove preinclude system compilers
-RUN rpm -e --nodeps --verbose gcc gcc-c++
-
 # Add custom `yum_clean_all` script before using `yum`
 COPY scripts/yum_clean_all /opt/docker/bin/
 
 # Fallback to CentOS vault for CentOS 8 support.
-RUN if [ "$DISTRO_VER" -eq "8" ]; then \
+RUN if [ "${DISTRO_NAME}${DISTRO_VER}" = "centos8" ]; then \
         find /etc/yum.repos.d/ -name "CentOS-*.repo" -exec \
              sed -i 's/mirrorlist/#mirrorlist/g;s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' {} \; && \
         yum update -y --disablerepo=cuda && \
         /opt/docker/bin/yum_clean_all; \
     fi
 
+# Remove preinclude system compilers
```

Comment:
I moved this command down, to be consistent with the same change for aarch64 in #215.

Reply:
Yeah that makes. Didn't mean to have the change slip into that PR

That said, we should probably do this after all `yum update` commands

Reply:
Done in 93799ede3e175a5ca9e3681e5b475fb7436bc053.